codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/PaginatorGenerator.kt

```diff
@@ -139,13 +140,21 @@ class PaginatorGenerator private constructor(
                     Self {
                         handle,
                         builder,
+                        stop_on_duplicate_token: true,
                     }
                 }
 
                 #{page_size_setter:W}
 
                 #{items_fn:W}
 
+                /// Stop paginating when the service returns the same pagination token twice in a row.
+                ///
```

Comment:
We should include docs on when this would be helpful to set to false:
> When running status-checking operations or operations that you want to continually send in order to "watch" for the latest data (like AWS CloudWatch's `GetLogEvents`), set this to false. That way, you can continually call the paginator in order to receive new data as it becomes available.